.devcontainer/devcontainer.json

```diff
@@ -38,6 +47,50 @@
       "!include_dir_list scalar",
       "!include_dir_merge_list scalar",
       "!include_dir_merge_named scalar"
+    ],
+    // python common
+    "[python]": {
+      "editor.defaultFormatter": "ms-python.python",
+      "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+      }
+    },
+    "python.linting.enabled": true,
```

Comment:
This is already the default and unneeded

Reply:
Refer to answer: https://github.com/home-assistant/core/pull/73940#discussion_r906002128